lib/screens/login/login_repository.dart

```diff
@@ -12,14 +15,50 @@ class FakeLoginRepository implements LoginRepository {
   @override
   Future<String> login(String email, String password) async {
     // Simulate network delay
+    SharedPreferences sharedPreferences = await SharedPreferences.getInstance();
+
     await Future.delayed(Duration(seconds: 1));
 
     if (Random().nextBool()) {
       // random network error
       throw NetworkException();
     } else {
       // fake successful login
-      return "Token abcdEfgh";
+      sharedPreferences.setString(S.auth, "Token 9935a8b04f2de7f5dec8f9e92a1893822b034dc7");
+      return "Token 9935a8b04f2de7f5dec8f9e92a1893822b034dc7";
+    }
+  }
+}
+
+class APILoginRepository implements LoginRepository {
+  @override
+  Future<String> login(String email, String password) async {
+    http.Response response;
+    SharedPreferences sharedPreferences = await SharedPreferences.getInstance();
+    try {
+      response = await http.post(
+        S.loginUrl,
+        body: <String, dynamic>{S.emailKey: email, S.passwordKey: password},
```

Comment:
:red_circle:
in `strings.dart`, you have declared `emailKey` and `passwordKey` as **SharedPref Keys** but are using it here as **API keys**. Also, the values you have assigned to them are not the same as given in api documentation. Please rectify.